### DISCUSSION TITLE
阿里和腾讯DNS的Dot和DoH现状如何？稳不稳？

### DISCUSSION THREAD

**Discussion by PHCSJC** [IrineSistiana/mosdns#459] [Q&A]
就是想了解下，解析稳不稳定，会不会经常失败？

个人也用了好久了，好像没什么问题，就是想再深入交流下国内Dot和DoH的现状

**Comment by qingtian110** [comment]
我在用腾讯的专业解析  走的 DoH    个人感觉良好
